openslides/motions/static/templates/motions/motion-list.html

```diff
@@ -21,78 +21,184 @@ <h1 translate>Motions</h1>
 
 <div class="row form-group">
   <div class="col-sm-8">
-    <!-- TODO: add filters -->
+    <form class="form-inline">
+      <!-- delete mode -->
+      <div os-perms-lite="motions.can_manage" class="form-group">
+        <label for="deleteSwitcher" translate>Delete mode</label>
+        <switch id="deleteSwitcher" ng-model="isDeleteMode" ng-change="uncheckAll()"
+            on="{{'On'|translate}}" off="{{'Off'|translate}}"
+            class="green wide form-control">
+        </switch>
+      </div>
+      <!-- delete button -->
+      <a ng-show="isDeleteMode && (motions|filter:{selected:true}).length > 0"
+          os-perms="motions.can_manage" ng-click="delete()"
+          class="btn btn-primary btn-sm form-control">
+        <i class="fa fa-trash fa-lg"></i>
+        <translate>Delete selected motions</translate>
+      </a>
+      <!-- state filter -->
+      &nbsp;
+      <select ng-model="stateFilter" class="form-control" id="stateFilter">
+        <option value="" translate>--- Select state ---</option>
+        <!-- TODO: use default workflow id from config('motions_workflow') -->
```

Comment:
The fix can wait until we rework the workflow api for 2.1. Then only one workflow with all states is shown in the select box. For 2.0 we can use this filter which contains all states group-by workflows. So done here.